test/Microsoft.Framework.Configuration.FunctionalTests/ConfigurationTests.cs

```diff
@@ -236,6 +236,37 @@ public void SetBasePathCalledMultipleTimesForEachSource()
             File.Delete(_jsonConfigFilePath);
         }
 
+        [Fact]
+        public void GetDefaultBasePathForSources()
+        {
+            // Arrange
+            var builder = new ConfigurationBuilder();
+            _jsonConfigFilePath = Path.Combine(Directory.GetCurrentDirectory(), "test.json");
```

Comment:
Not sure we should use the _jsonConfigFilePath field here as opposed to just a local variable. It seems that most tests could be relying on things being setup the way they are setup in the `ConfigurationTests` constructor and if we drift from that we could cause random failures. I think the same applies to the other test that you added recently: `SetBasePathCalledMultipleTimesForEachSource()`. Perhaps we should make those fields read-only?